lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.ts

```diff
@@ -549,4 +551,17 @@ export class ContentMetadataComponent implements OnChanges, OnInit {
         }
         return observables;
     }
+
+    private updateInvalidProperties() {
+        if (this.targetProperty?.isValidValue === false) {
```

Comment:
Do we really need to check boolean values like === false and === true? Can't it be like for example: 
if (this.targetProperty?.isValidValue)
?

Reply:
I added this statement on purpose as an extra precaution step